### ISSUE TITLE
iOS is not sending events

### ISSUE THREAD

**Issue by reeq-dev** [mixpanel/mixpanel-react-native#190]
iOS does not track any events even from freshly installed project. 0 logs after `.setLoggingEnabled(true); ` from xcode or rn terminal.
Since the react-native package installs a swift package as well I tried to setup and fire event from swift file and it worked - so problem is related to react-native wrapper
Android seems fine.

XCode: 14.2 (14C18)
Chip: M1 Pro
"react-native": "0.71.7"
"mixpanel-react-native": "^2.2.4",

Pay attention that there is some changings in Podfile to allow launch app on m1:
```
    installer.pods_project.build_configurations.each do |config|
      config.build_settings["EXCLUDED_ARCHS[sdk=iphonesimulator*]"] = "arm64"
    end
```
and excluded archs in build settings for target: arm64 aswell

repo to reproduce
https://bitbucket.org/nreeq/mixpanel_test_project/src/master/

**Comment by reeq-dev** [comment]
Update: downgrade for 2.2.1 made it work

**Comment by Dallas62** [comment]
Experiencing same issue, drop of mixpanel events since last update.

**Comment by hilat-autofleet** [comment]
experiencing same issue, downgraded to 2.2.1

**Comment by nayanzin3sergey** [comment]
I'm getting the same issue.. any idea how to resolve? already downgrade it to 2.2.1

**Comment by alimomos** [comment]
experiencing same issue, downgraded to 2.2.1 doesn't work as well.. any other suggestions?

**Comment by hilat-autofleet** [comment]
> experiencing same issue, downgraded to 2.2.1 doesn't work as well.. any other suggestions?

1. remove the app from device 
2. npm install & pod install
3. clean build 
4. build 
5. reinstall the app.

**Comment by alimomos** [comment]
Hey @hilat-autofleet , yes It works .. forgot to update here. btw thanks 🙏

**Comment by gimi-anders** [comment]
It looks like the issue has been solved in version 2.2.5. At least I am seeing events from iOS on that version.

**Comment by jaredmixpanel** [comment]
This issue should be fixed in v2.2.5. Please update and let us know if the issue persists.
